src/styles/Shared/Header/_PrimaryNav.scss

```
@@ -26,6 +26,9 @@
       .easya-btn-nav {
         margin: -4px 0;
         cursor: pointer;
+        &:hover{
+          color: $tango;
+        }
```

Make sure you use the prettier settings... Just add a space:

```suggestion
        &:hover {
          color: $tango;
        }
```